build.gradle

```diff
@@ -7,11 +7,14 @@ buildscript {
     }
 }
 
+group = 'io.reactivex.rxjava2'
+
+
 apply plugin: 'nebula.rxjava-project'
 apply plugin: 'kotlin'
 
 dependencies {
-    compile 'io.reactivex.rxjava2:rxjava:2.0.5'
+    compile 'io.reactivex.rxjava2:rxjava:2.0.7'
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     testCompile 'org.funktionale:funktionale:0.9'
```

Comment:
This one should be updated to `org.funktionale:funktionale-partials:1.0.0-final`.